src/locale/locales/zh-CN/messages.po

```diff
@@ -19,69 +19,69 @@ msgstr "(没有邮件)"
 
 #: src/view/com/notifications/FeedItem.tsx:239
 msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
-msgstr ""
+msgstr "其他 {formattedCount} 人"
 
 #: src/components/moderation/LabelsOnMe.tsx:57
 msgid "{0, plural, one {# label has been placed on this account} other {# labels has been placed on this account}}"
-msgstr ""
+msgstr "{0} 个标签已应用到此账户"
```

Comment:
@quiple @tkusano Hi there, since East Asian languages don't have plural forms, I think we can simplify it to this form. What do you think? It works on Simplified Chinese.

Reply:
Oh, I don't know why, but I noticed that the commas in the numbers disappeared (like `Liked by 7,015 users`) when I typed it this way.

It's probably better to write it like `{0, plural, other {#}}`.

Reply:
Thanks for your feedback, this is indeed an issue. Until I find the specific reason, I will use `{0, plural, other {#}}`.

Reply:
@auroursa @quiple 

![image](https://github.com/bluesky-social/social-app/assets/30096511/097c7bc4-022a-496d-96f0-0c7ed041d2fd)


Try this it might be work

Reply:
Update this is wrong

Reply:
@quiple can you show me which case or post can find `Liked by 7,015 users` I search all around and no where can i find a case it will show

Reply:
@auroursa @quiple I have been viewed carefully and viewed the code and find out the truth

![image](https://github.com/bluesky-social/social-app/assets/30096511/29723029-a6aa-4e3a-b022-01fb37e871eb)

The former({0}) is used as a number, while the latter({likeCount}) is a string, apply it by the logic

this version will work!!!!!!!